sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -1906,6 +1999,14 @@ private bool IsExcluded(Cursor cursor, out bool isExcludedByConflictingDefinitio
                 return false;
             }
 
+            if (_config.ExcludeFunctionsWithBody &&
```

Comment:
`IsExcluded` is actually one of the "hottest" functions in ClangSharp. It would probably be good to keep this after the `IsExcludedByFile` check which catches most of the exclusions.